### DISCUSSION TITLE
Project Pythia Email List

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/projectpythia.github.io#112] [General]
If you would like join the projectpythia@ucar.edu Google group to be informed of updates and events in the seminar series, please do so by following this [link](https://groups.google.com/a/ucar.edu/g/projectpythia/about) and clicking "Join Group" next to the group name.

Should we add this to the contributor's guide? Should we move all meetings to be a part of this Google Group calendar?

**Comment by erinmr** [comment]
Hi @jukent - When I follow that [link](https://groups.google.com/a/ucar.edu/g/projectpythia/about) I get an access error. I would like to be added to the list, if possible (erinmr@gmail.com). Thanks!

**Reply by jukent** [reply]
Added!

**Comment by brian-rose** [comment]
@jukent sounds like we need to fix the link? Or does the google group not actually exist yet? I can't seem to find the group by following that link either.

**Reply by jukent** [reply]
I believe the Google group does exist, because my test email to projectpythia@ucar.edu worked. I wonder if it is only visible to UCAR internal people though. @kmpaul @clyne could you test the link: https://groups.google.com/a/ucar.edu/g/projectpythia/about

**Reply by andersy005** [reply]
@jukent, I don't think this group is publicly accessible for those with emails outside the `ucar.edu` domain ( I tried accessing it/ searching it with my own email address without success)

**Reply by clyne** [reply]
> I believe the Google group does exist, because my test email to [projectpythia@ucar.edu](mailto:projectpythia@ucar.edu) worked. I wonder if it is only visible to UCAR internal people though. @kmpaul @clyne could you test the link: https://groups.google.com/a/ucar.edu/g/projectpythia/about

Test it how? I can open the link if that is what you mean.

**Reply by jukent** [reply]
That is what I meant. Sounds like we have correctly identified the problem. I will talk to the admins to see if we can fix this.

**Reply by jukent** [reply]
Okay this should be fixed now. Can someone verify?

**Comment by erinmr** [comment]
It looks like the settings are anyone in the org (UCAR) can ask to join the group. Maybe anyone on the web? 
![Screen Shot 2021-06-29 at 4 44 03 PM](https://user-images.githubusercontent.com/2915555/123876858-533e9a00-d8f9-11eb-94a0-cf2c0e7732f9.png)

**Reply by erinmr** [reply]
also please delete that screenshot if needed.

**Reply by jukent** [reply]
Thanks I will try to fix this tomorrow.

**Reply by jukent** [reply]
I have requested to change the "can see group" and "can ask to join the group" settings to anyone on the web. This might take a few days for CISL IT Support to resolve.

**Reply by jukent** [reply]
Done.
